avocado/utils/pci.py

```diff
@@ -53,7 +53,7 @@ def get_pci_addresses():
     addresses = []
     cmd = "lspci -D"
     for line in process.system_output(cmd).splitlines():
-        if "PCI bridge" not in line:
+        if "PCI bridge" not in line and "peripheral" not in line:
```

Comment:
What the `peripheral` filter stands for? I don't have it in my `lspci -D` (x86, arm, ppc)